Z/ZeroMQ/build_tarballs.jl

```diff
@@ -18,6 +18,9 @@ if [[ "${target}" == *86*-linux-musl* ]]; then
     atomic_patch -d /opt/${target}/lib/gcc/${target}/*/include -p0 $WORKSPACE/srcdir/patches/mm_malloc.patch
 fi
 sh autogen.sh
+echo "=====configure====="
+cat configure
+echo "=====configure====="
```

Comment:
You need to wait for https://github.com/JuliaRegistries/General/pull/122028 to be merged.

Reply:
Ah, okay. I saw the binary uploaded to the binary wrappers repo and thought it would pick it up from there. What is the change that had to get made to Compat.toml for passing the registry consistency check?

I'll revert this commit.